src/controllers/router-store/index.tsx

```diff
@@ -102,6 +109,7 @@ const actions: AllRouterActions = {
       const { route, match, query } = getState();
```

Comment:
What is the future of `requestRouteResources` action? Is this superseded by one of the plugin methods? Can it be deleted?